api/api-doc/storage_service.json

```diff
@@ -104,6 +104,54 @@
             }
          ]
       },
+      {
+         "path":"/storage_service/toppartitions/",
+         "operations":[
+            {
+               "method":"GET",
+               "summary":"Toppartitions query",
+               "type":"toppartitions_query_results",
+               "nickname":"toppartitions_generic",
+               "produces":[
+                  "application/json"
+               ],
+               "parameters":[
+                  {
+                     "name":"name_list",
```

Comment:
name_list is not a good name, it's either cf or table if you want to go with the current names.
Because it's a new method, why not breaking it to two parameters, one for keyspace and one for table (ks and cf/table)

Reply:
By two methods you mean one for keyspace filters and one for families in the current format ("ks:cf")?

Reply:
this is a new method, I'm saying that you can use two parameters, i.e. one for keyspace and one for cf